### PR TITLE
Update remove-former-employee-step-1.md

### PR DESCRIPTION
**Pull request by robertkruk**
Updated steps to align more with "the New Exchange admin center"

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/add-users/remove-former-employee-step-1.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ If you have email as part of your Microsoft 365 subscription, sign in to the Exc
   
 1. Go to the <a href="https://admin.exchange.microsoft.com/" target="_blank">Exchange admin center</a>.
 2. In the Exchange admin center, navigate to **Recipients** \> **Mailboxes**.
-3. Double-click the user and go to **Manage email apps settings** under **Email apps**. Turn **Off** the slider for all the options; **Mobile (Exchange ActiveSync)**, **Outlook on the web**, **Outlook desktop (MAPI)**, **Exchange web services**, **POP3**, and **IMAP**.
+3. Select the user mailbox from the list and then in the *Details Pane* (on the right hand side), click on **Manage email apps settings** under **Email apps**. Turn **Off** the slider for all the options; **Mobile (Exchange ActiveSync)**, **Outlook on the web**, **Outlook desktop (MAPI)**, **Exchange web services**, **POP3**, and **IMAP**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-1.md]:
```suggestion
3. Select the user mailbox from the list and then, in the *Details Pane* (on the right-hand side), select **Manage email apps settings** under **Email apps**. Turn **Off** the slider for all the options; **Mobile (Exchange ActiveSync)**, **Outlook on the web**, **Outlook desktop (MAPI)**, **Exchange web services**, **POP3**, and **IMAP**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @robertkruk, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**